ovos_core/intent_services/converse_service.py

```diff
@@ -48,6 +48,7 @@ def deactivate_skill(self, skill_id, source_skill=None):
                 self.active_skills.pop(idx)
                 self.bus.emit(
                     Message("intent.service.skills.deactivated",
+                            {"skill_id": skill_id},
```

Comment:
I'm sure this is correct that this is duplicated to both data and context? but maybe to be clear use data=..., context=...
?
(same a bit lower...)

Reply:
in ovos we add skill_id to the context of all bus messages, these were missing in a couple places i noticed during hivemind testing (which meant LocalHive was not mapping these to the correct skill to route messages)

probably could go into a different PR, just had this branch checked out while working on it